docs/config.rst

```diff
@@ -234,6 +235,17 @@ Features
 
    .. versionadded:: 2.0
 
+.. attribute:: LP_ENABLE_CONTAINER
+   :type: bool
+   :value: 0
+
+   Indicate if the shell is running in a container environment (e.g.
+   Docker, Podman, LXC, Singularity)
+
+   See also: :attr:`LP_COLOR_CONTAINER`.
+
+   .. versionadded: 2.0
```

Comment:
This will be merged after v2.0, so change these to 2.1.

Reply:
Fixed